### ISSUE TITLE
Error (mount.c:1254): mnt: Can't restore tmpfs content

### ISSUE THREAD

**Issue by eliams** [xemul/criu#32]
Hi,
I am using experimental docker with C/R commands but failed to restore a container.

My environment:
Debian based with custom 4.0.0 kernel.
Docker version 1.9.0-dev, build 2919249, experimental
  (I am using aufs as storage driver)
CRIU Version: 1.7 GitID: v1.7-42-g9353051

Daemon logs: http://pastebin.com/vEhtpb97
Restore logs: http://pastebin.com/y5kjmEb1

I have another host (let's call it A) with the same environment on which C/R works fine so I don't understand why it's falling on host B.
Thanks for the help and let me know if I can provide any other information.

**Comment by xemul** [comment]
Can you check whether the fresh tree with 84cc1e12ac009b41395f1015202b1f90762abe06 fixes the issue?

**Comment by eliams** [comment]
Yes it fixed the bug, restoring now work on host B (for a container created on host B). Thanks !

However my goal is to checkpoint a container on one host and restore it in the other by moving the image directory and using --force option to restore in another container.
So I created a container on host B but restoring failed on host A.
What I did:
Host B:
docker run -d busybox /bin/sh -c 'i=0; while true; do echo $i; i=$(expr $i + 1); sleep 1; done'  
docker checkpoint --work-dir=<work-dir> --image-dir=<image-dir> <container-id>

Host A:
docker create busybox
docker restore --work-dir=<work-dir> --image-dir=<folder copied from host B> --force=true <created container id>

restore logs: http://pastebin.com/jpnhuePH
I don't understand if the /bin/sh mentioned in the error is from the containers or the hosts.

Thanks again for the help !
PS: Let me know if it's better to create a new issue for this problem.
